src/Search/Provider/SQLProvider.php

```diff
@@ -92,75 +102,71 @@ public static function addDefaultSelect($itemtype)
             $item           = getItemForItemtype($itemtype);
             $mayberecursive = $item->maybeRecursive();
         }
-        $ret = "";
+        $ret = [];
         switch ($itemtype) {
             case 'FieldUnicity':
-                $ret = "`glpi_fieldunicities`.`itemtype` AS ITEMTYPE,";
+                $ret[] = "`glpi_fieldunicities`.`itemtype` AS ITEMTYPE";
```

Comment:
You should try to replace all these backticks (`) by `$DB::quoteName()`, when you refactor a line that contains it.

Reply:
I'm working on another PR to remove hard-coded quotes for names and values.